lighthouse-core/gather/computed/trace-of-tab.js

```diff
@@ -107,7 +114,7 @@ class TraceOfTab extends ComputedArtifact {
       firstPaint,
       firstContentfulPaint,
       firstMeaningfulPaint,
-      traceEnd,
+      traceEnd: latestTraceEvt,
```

Comment:
I had a very good explanation for why I named them this instead of traceEnd (that these were supposed to be just about the main thread, `latestMainThreadEvt` `earliestMainThreadEvt` and distinct from `traceEnd`) but the rest of the code betrays me. In fact, there are a number of things in here that I strongly feel like I accidentally dropped a stash...